### PR TITLE
stm32l432 support

### PR DESCRIPTION
**Pull request by geoffreymbrown**
Changes to support stm32l432

### CODE REVIEW

File: include/stlink/chipid.h [hunk 1]
```diff
@@ -54,6 +55,9 @@ enum stlink_stm32_chipids {
 	STLINK_CHIPID_STM32_F7               = 0x449,
 	STLINK_CHIPID_STM32_F7XXXX           = 0x451,
 	STLINK_CHIPID_STM32_F410             = 0x458
+
+
+
```
Comment (xor-gate) [inline include/stlink/chipid.h]: Please dont add whitespaces

File: src/chipid.c [hunk 2]
```diff
@@ -406,6 +406,22 @@ static const struct stlink_chipid_params devices[] = {
             .bootrom_base = 0x1fff0000,      // Tables 4-6, pages 80-81 (Bank 1 system memory)
             .bootrom_size = 0x7000           // 28k (per bank), same source as base
         },
+	        {
+            // 	STLINK_CHIPID_STM32_L43X   
+            // From RM0392.
+ 	    .chip_id = STLINK_CHIPID_STM32_L43X,   
+            .description = "L4 device",
```
Comment (xor-gate) [inline src/chipid.c]: L43x device

**Comment by geoffreymbrown** [comment]
Ok.  Do you want me to correct the whitespace ?  Or is it easier for you to
handle ?
Geoffrey

**Comment by geoffreymbrown** [comment]
ok, I've pushed the changes -- looks like they show up in the pull request.
Geoffrey

**Comment by xor-gate** [comment]
Yes all changes in a separate branch go into the same pull request. Thank you!

**Pull request merged**